riotfile.py

```diff
@@ -129,22 +129,26 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             ],
         ),
         Venv(
-            pys=["3"],
+            pys=["3.9"],
             name="mypy",
             command="mypy {cmdargs}",
             create=True,
             pkgs={
-                "mypy": latest,
+                "mypy": "==0.961",
```

Comment:
Does latest not work?

Reply:
If we set the latest version we could have "surprise" in the CI, what do you think? I can rollback this change and discuss it later  :upside_down_face: